lib/IMAP/MailboxSync.php

```diff
@@ -0,0 +1,141 @@
+<?php declare(strict_types=1);
```

Comment:
newline?

Reply:
this was intentional, as explained above